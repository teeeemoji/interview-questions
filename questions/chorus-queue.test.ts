import chorusQueue from './chorus-queue'

test('186 186 150 200 160 130 197 200',  () => {
  expect(chorusQueue([186,186,150,200,160,130,197,200])).toMatchSnapshot()
})