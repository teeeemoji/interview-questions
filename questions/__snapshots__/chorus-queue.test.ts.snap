// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`186 186 150 200 160 130 197 200 1`] = `0`;
